### ISSUE TITLE
Better manual

### ISSUE THREAD

**Issue by gsedej** [kokoko3k/ssh-rdp#15]
Hello. I this project very interesting when trying to create "remote gaming guest" in proxmox.

The issues I had is that "netevent" dependency was not obvious. Ubuntu does not have it in repo, also when searching "netevent" with duckduckgo, you don't get  results. Maybe it should be mentioned to get it on https://github.com/Blub/netevent and compile it.

The "inputconfig" and "grab hotkey" parts are somehow not enough visible. Maybe change "Usage:" to "Client setup" or something similar?

_(also the mvp in 18.04 does not have flag --profile=low-latency )_

Anyway thank you for this very interesting project!

**Comment by kokoko3k** [comment]
Hello,
I updated the README with a link to netevent.

About inputconfig, is literally the first instruction in the readme:

```
Usage:

    To make it work, first start it with the "inputconfig" option
```
What do you mean exactly?

About mpv, from git log, it seems low-latency profile has been added in March, 2018, i'll add a note about the needed mpv v0.29.0 version

**Comment by kokoko3k** [comment]
Closing because no reply, but feel free to ask to reopen if you're still interested.

**Comment by ntropia2** [comment]
Great project! 

I second the suggestion of improving the documentation.
One thing that I found not so obvious is the distinction of the roles between local and remote, e.g.: how to configure a scenario in which the remote server is the one where a game runs and the local one is the client. Also, it's not obvious from the get-go how to interact with the remote server, for example.

Also, adding a few usage cases scenarios (gaming being arguably the most obvious and popular) could get more interest. For example, I've bumped into this project following a comment on [Soylent News](https://soylentnews.org/comments.pl?noupdate=1&sid=56061&page=1&cid=1311985#commentwrap), which made it sound incredibly appealing so I tried it (and wasn't disappointed!)
